go/apps/api/openapi/openapi-generated.yaml

```
@@ -482,6 +639,33 @@ components:
                 pagination:
                     "$ref": "#/components/schemas/Pagination"
             additionalProperties: false
+            example:
+                meta:
+                    requestId: req_1234567890abcdef
+                data:
+                    - externalId: user_1234567890abcdef
+                      meta:
+                        plan: premium
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 26 but found 24

(indentation)